cluster-api/src/main/java/io/aiven/klaw/clusterapi/controller/ClusterApiController.java

```diff
@@ -0,0 +1,252 @@
+import io.aiven.klaw.clusterapi.models.AclsNativeType;
+import io.aiven.klaw.clusterapi.models.ApiResponse;
+import io.aiven.klaw.clusterapi.models.ClusterAclRequest;
+import io.aiven.klaw.clusterapi.models.ClusterSchemaRequest;
+import io.aiven.klaw.clusterapi.models.ClusterStatus;
+import io.aiven.klaw.clusterapi.models.ClusterTopicRequest;
+import io.aiven.klaw.clusterapi.models.KafkaSupportedProtocol;
+import io.aiven.klaw.clusterapi.services.AivenApiService;
+import io.aiven.klaw.clusterapi.services.ApacheKafkaAclService;
+import io.aiven.klaw.clusterapi.services.ApacheKafkaTopicService;
+import io.aiven.klaw.clusterapi.services.MonitoringService;
+import io.aiven.klaw.clusterapi.services.SchemaService;
+import io.aiven.klaw.clusterapi.services.UtilComponentsService;
+import java.util.*;
+import javax.validation.Valid;
+import lombok.AllArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.http.*;
+import org.springframework.web.bind.annotation.*;
+
+@RestController
+@RequestMapping("/topics")
+@Slf4j
+@AllArgsConstructor
+public class ClusterApiController {
+
+  UtilComponentsService utilComponentsService;
+
+  ApacheKafkaAclService apacheKafkaAclService;
+
+  ApacheKafkaTopicService apacheKafkaTopicService;
+
+  SchemaService schemaService;
+
+  MonitoringService monitoringService;
+
+  AivenApiService aivenApiService;
+
+  @RequestMapping(
+      value = "/getApiStatus",
+      method = RequestMethod.GET,
+      produces = {MediaType.APPLICATION_JSON_VALUE})
+  public ResponseEntity<ClusterStatus> getApiStatus() {
+    return new ResponseEntity<>(ClusterStatus.ONLINE, HttpStatus.OK);
+  }
+
+  //    @RequestMapping(value = "/reloadTruststore/{protocol}/{clusterName}", method =
+  // RequestMethod.GET,produces = {MediaType.APPLICATION_JSON_VALUE})
+  //    public ResponseEntity<String> reloadTruststore(@PathVariable String protocol,
+  //                                                   @PathVariable String clusterName){
+  //        return new ResponseEntity<>(manageKafkaComponents.reloadTruststore(protocol,
+  // clusterName), HttpStatus.OK);
+  //    }
+
+  @RequestMapping(
+      value = "/getStatus/{bootstrapServers}/{protocol}/{clusterName}/{clusterType}",
+      method = RequestMethod.GET,
+      produces = {MediaType.APPLICATION_JSON_VALUE})
+  public ResponseEntity<ClusterStatus> getStatus(
+      @PathVariable String bootstrapServers,
+      @Valid @PathVariable KafkaSupportedProtocol protocol,
+      @PathVariable String clusterName,
+      @PathVariable String clusterType) {
+    return new ResponseEntity<>(
+        utilComponentsService.getStatus(bootstrapServers, protocol, clusterName, clusterType),
+        HttpStatus.OK);
+  }
+
+  @RequestMapping(
+      value = "/getTopics/{bootstrapServers}/{protocol}/{clusterName}",
+      method = RequestMethod.GET,
+      produces = {MediaType.APPLICATION_JSON_VALUE})
+  public ResponseEntity<Set<HashMap<String, String>>> getTopics(
```

Comment:
```suggestion
  public ResponseEntity<Set<Map<String, String>>> getTopics(
```
is not just `Map` enough here?